UVAtlas/pch.h

```diff
@@ -102,3 +114,12 @@ extern void __cdecl UVAtlasDebugPrintf(unsigned int lvl, _In_z_ _Printf_format_s
 #ifndef SAFE_RELEASE
 #define SAFE_RELEASE(p)      { if(p) { (p)->Release(); (p)=nullptr; } }
 #endif
+
+// HRESULT_FROM_WIN32(ERROR_ARITHMETIC_OVERFLOW)
```

Comment:
``HRESULT_FROM_WIN32`` and the associated ``ERROR_`` values are not available in the ``winadapter.h`` on *DirectX-Headers*.